com.warlordsoftwares.formatlab.yml

```diff
@@ -0,0 +1,169 @@
+app-id: com.warlordsoftwares.formatlab
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: formatlab
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
+  - --filesystem=xdg-download
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-run/gvfsd
+  - --device=dri
+  - --socket=pulseaudio
+  - --env=QT_QPA_PLATFORM=wayland
```

Comment:
```suggestion
```
It won't work on x11

Reply:
does it actually support wayland?

Reply:
it is tested on wayland

Reply:
again this won't work with X11.

Reply:
without this flag my application will not run. it is giving error and not starting.

Reply:
even when flag is set to x11 . it won't open

Reply:
`x11` isn't a valid value.

But you don't seem to understand what `QT_QPA_PLATFORM` does. Here it forces wayland which will fail on X11.